src/test/java/no/nav/familie/integrasjoner/oppgave/OppgaveControllerTest.kt

```diff
@@ -320,6 +319,56 @@ class OppgaveControllerTest : OppslagSpringRunnerTest() {
         assertThat(response.statusCode).isEqualTo(HttpStatus.BAD_REQUEST)
     }
 
+    @Test
+    fun `finnOppgaverV2 skal fungere ved retur av 0 oppgaver`() {
+        stubFor(get("/api/v1/oppgaver?statuskategori=AAPEN&tema=BAR&sorteringsfelt=OPPRETTET_TIDSPUNKT&sorteringsrekkefolge=DESC&limit=50&offset=0")
+                .willReturn(okJson(gyldigOppgaveResponse("tom_response.json"))))
+
+        val response: ResponseEntity<Ressurs<FinnOppgaveResponseDto>> =
+                restTemplate.exchange(localhost("/api/oppgave/v2"), HttpMethod.POST, HttpEntity(FinnOppgaveRequest("BAR"), headers))
+
+        assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
+        assertThat(response.body?.data?.oppgaver).isEmpty()
```

Comment:
response.body?.data?.antallTreffTotalt should also be tested, so is line 343, line 361.